pkg/matcher/annotation_matcher.go

```diff
@@ -148,7 +148,7 @@ func MatchPipelinerunByAnnotation(ctx context.Context, logger *zap.SugaredLogger
 			prMatch.Config["target-namespace"] = targetNS
 			prMatch.Repo, _ = MatchEventURLRepo(ctx, cs, event, targetNS)
 			if prMatch.Repo == nil {
-				logger.Warnf("could not find Repository CRD in %s while pipelineRun %s targets it", targetNS, prun.GetGenerateName())
+				logger.Warnf("could not find Repository CRD in branch %s, the pipelineRun %s has a label that explictely targets it", targetNS, prun.GetGenerateName())
```

Comment:
stuppid mispell 
```suggestion
				logger.Warnf("could not find Repository CRD in branch %s, the pipelineRun %s has a label that explicitly targets it", targetNS, prun.GetGenerateName())
```

Reply:
reviewed your own code 😄 cool!

Reply:
I was lazy to do a git push --force again :D